Assignment/solution_student/css/style.css

```diff
@@ -1,27 +1,193 @@
 /*
 Project Name: HTML Template
 Client: Your Client
-Author: Your Name
+Author: Carly Brandess
 Developer: @poopsplat in Chicago
 */
 
-/******************************************
-/* BASE STYLES
+/* SETUP
 /*******************************************/
+/* Box Model Hack: http://www.paulirish.com/2012/box-sizing-border-box-ftw/ */
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+html {
+  box-sizing: border-box;
+}
+/* Micro clearfix hack: http://nicolasgallagher.com/micro-clearfix-hack/ */
+.clearfix {
+  *zoom: 1;
+}
+.clearfix:before,
+.clearfix:after {
+    content: " ";
+    display: table;
+}
+.clearfix:after {
+  clear: both;
+}
+
+a:hover {
+    text-decoration: none;
+}
+
+header { 
+ 	background-image: url("../images/header_bg.jpg");
+	width: 100%;
+	height: 500px;
+	background-size: contain;
+	background-repeat: no-repeat;
+	margin: 0;
+  	padding: 0;
+}
+
+h1 {
+	color: white;
+	font-weight: bolder;
+	font-size: 24px;
+	padding-left: 20px;
+	padding-top: 5px;
+}
+
+h2 {
+	color: white;
+	font-weight: semibold;
+	font-size: 14px;
+	border-radius: 5px;
+	text-align: center;
+	padding-top: 100px;
+}
+
+h3 {
+	color: #033048;
+	font-size: 20px;
+	font-weight: semibold;
+	padding: 0;
+	margin: 0;
+}
+
+.container {
+	padding: 0;
+	padding-bottom: 20px;
+	margin: 0;
+	width: 50%;
+	text-indent: 20px;
+}
+
+li {
+	text-indent: 20px;
+}
+
+.nowbutton {
+	text-align: center;
+}
+
+.now{
+	position: relative;
+	top: 75%;
```

Comment:
This `top` property isn't working here because it's an inline element, and if you recall our last lesson on display, inline elements can't get a height, and similarly they can't be positioned by their `top` or `bottom` properties. It's not needed here anyway however, if you want to space it away from the text above it, use `margin-top`.